README.md

```diff
@@ -161,6 +161,19 @@ Generate a script that points KUBECONFIG to the targeted cluster for the specifi
 eval $(gardenctl kubectl-env bash)
 ```
 
+To load the kubectl configuration for each bash session add the following line at the end of the `~/.bashrc` file:
+
+```bash
+eval "$(gardenctl kubectl-env bash)"
+```
```

Comment:
the command is duplicative to the previous command (which should be fixed to `eval "$(gardenctl kubectl-env bash)"`). Instead, I would suggest to just refer to the previous command instead of repeating it

Reply:
Fixed via [`4849029` (#490)](https://github.com/gardener/gardenctl-v2/pull/490/commits/48490295b403fccae0943a348caf5c0a3e47bd55) & [`f873fc9` (#490)](https://github.com/gardener/gardenctl-v2/pull/490/commits/f873fc9e63810414eb1e9001feeb7f59708820fa)